### ISSUE TITLE
Update common entities

### ISSUE THREAD

**Issue by jphall663** [responsible-ai-collaborative/aiid#2590]
I suggest we update the common entities feature on the incidentdatabase.ai landing page. It would be ideal for the common entities to reflect systems instead of/in addition to companies/organizations. 

I understand this could require improved query mechanisms, and we may consider minimizing the feature until more incident response occurs or query approaches can be updated to reflect systems.

**Comment by kepae** [comment]
I agree. The entities we currently show rarely change, which doesn't give a sense of the times (e.g. LLM chatbot chaos).

There's a few things we can do.

* In the short term, minimize the common entities feature. Move it down the page, or remove entirely?
    * Or, calculate the most common entities in the recent N months (3? 6?) to provide a more recent view. We can calculate what this would be and see.
* In the longer term, support indexing AI systems as entities in our entity collection. We currently only attempt to name developers, deployers, and victims – that is, organizations and people – and we don't characterize them beyond name. Given the challenges inherent to _stably naming and separating AI systems (or components)_ from one another, we would probably also prioritize:
    * #2465 
    * #2536 
    * As well as -- generalizing our entity relationships so that "links" between data on the platform are not restricted to particular documents (e.g. reports only point to incidents, but can also be linked to systems and other entities in a semantic way).

@jphall663 @datherton09 @Janetbananet -- this change and another one suggested (#2563) have editorial and voice impacts on the landing page. I would suggest we discuss these in the next product meeting.

**Comment by smcgregor** [comment]
Idea to throw in the mix: list the most recent incident responses.

**Comment by jphall663** [comment]
In the short term, I am in favor of:

> Move it down the page, or remove entirely

On #2563, my feeling is I like the current setup. It draws me in sometimes, however, it also has click-baity vibes. I am concerned that referencing just the incident will look boring. I wonder if we could keep an image in the display, as it is now, whether we link to incident or news report.

**Comment by Janet-ResponsibleAI** [comment]
If we move "Common Entities" down the page, then it bumps up the next item in the horizontal view which is the "Quick Add New Report" link. Not to say there is anything inherently wrong with that, but it points to a broader question about the contents and order of contents on the landing page.

I'm going to create a mock-up of some changes to the landing page and open a new GitHub issue to generate discussion regarding the overall content and order, with an eye towards what's above the fold.

Regarding the system entities: @kepae and I chatted briefly about this today as well. I had a few conversations with Khoa in the past about how complex in could be to 1. determine a standardized approach for naming systems, and 2. retroactively identify the system in each of the AIID's incidents (since this hasn't historically been part of the initial editorial process when ingesting and creating new incidents.)

**Comment by kepae** [comment]
Thanks for everyone's input! We need to make this element better, but as Janet pointed out, without accidentally making other elements more awkward. I'd like to experiment with making the top entities reflect a rolling window of "recent" entities before we delete it from the page – especially without an above-the-fold element to sensibly replace it.

Tagging @pdcp1 since we discussed this yesterday. Let's investigate the query that produces the element and experiment with 1-, 3-, and 6-month calculations of the recent implicated entities.

**Comment by kepae** [comment]
ps @Janet-ResponsibleAI for posterity, you had some prior thoughts on this here: https://github.com/responsible-ai-collaborative/aiid/issues/1491 !

**Comment by Janet-ResponsibleAI** [comment]
> ps @Janet-ResponsibleAI for posterity, you had some prior thoughts on this here: #1491 !

Whoa, thanks for digging this old one up! It's just what I need as a starting point.

**Comment by kepae** [comment]
@pdcp1 We had some conversation on this yesterday with Patrick, Janet, and co – it's most likely that we won't keep the "common entities" component at all, _unless_ we have some compelling data that shows that the entities change consistently when considering shorter windows of time (e.g. 1 month), rather than all time as it is now. Instead, we would be likely to highlight _incident responses_ in this spot, which is positive and complements the most recent incident report.

To help inform this home page UX decision, could you please
* calculate what the top-5 implicated entities are for three separate 1-month windows? Say November, December, and January?
* similarly, provide a timeline of incident responses received from November until $now? A simple list with dates would be fine here.

Then we can compare the two proposed changes.

**Comment by pdcp1** [comment]
@kepae This is the information required:

## Top-5 implicated entities for three separate 1-month windows

### Incidents of November 2023
<img width="1072" alt="November 2023" src="https://github.com/responsible-ai-collaborative/aiid/assets/6564809/ce716e06-6e7b-4bdd-8121-a6a956f35021">

### Incidents of December 2023
<img width="1069" alt="December 2023" src="https://github.com/responsible-ai-collaborative/aiid/assets/6564809/b8acadce-bd0c-4feb-bc8c-0c14e07556d1">

### Incidents of January 2024
<img width="1070" alt="January 2024" src="https://github.com/responsible-ai-collaborative/aiid/assets/6564809/593e90f4-4cb4-42e3-aee0-ed40dd30f0a3">

### Extra: Incidents from the last 3 months (Nov, Dec and Jan)
<img width="1167" alt="last 3 months" src="https://github.com/responsible-ai-collaborative/aiid/assets/6564809/de76d05f-6b86-40c6-9f41-8d27a77ddbb7">

## Incident responses received from November until now
[Incident responses from November 2023 until now.csv](https://github.com/responsible-ai-collaborative/aiid/files/14303221/Incident.responses.from.November.2023.until.now.csv)

**Comment by pdcp1** [comment]
@kepae @Janet-ResponsibleAI This is how the home page looks without the Common Entities block:

## Desktop
<img width="1800" alt="Homepage without Common Entities" src="https://github.com/responsible-ai-collaborative/aiid/assets/6564809/6fe127b3-c3df-46ec-93df-b97679d59fd4">

## Mobile (not affected)
![localhost_8000_(iPhone 12 Pro)](https://github.com/responsible-ai-collaborative/aiid/assets/6564809/008c7722-d1c2-4574-8c83-d65bfb5fb740)

**Comment by kepae** [comment]
Thanks for the demos! Let's go ahead with removing the common entities component, then we'll prototype the incident responses replacement after.

**Comment by kepae** [comment]
incidentdatabase.ai landing page now no longer has the common entities component. I'm lowering the priority on this issue now that we've "triaged" the page and while we determine other changes, perhaps worth discussing in the more specific context:
* #1491
